src/cases/containers/roles-and-access-container/roles-and-access-container.component.ts

```diff
@@ -19,17 +19,27 @@ export class RolesAndAccessContainerComponent implements OnInit {
   public showAllocateRoleLink: boolean = false;
   public locationInfo$: Observable<LocationInfo>;
 
-  constructor(private readonly route: ActivatedRoute, private readonly store: Store<fromCaseList.State>) {
+  constructor(private readonly route: ActivatedRoute,
+              private readonly store: Store<fromCaseList.State>,
+              private readonly appStore: Store<fromRoot.State>) {
   }
 
   public ngOnInit(): void {
     this.caseDetails = this.route.snapshot.data.case as CaseView;
+    const jurisdictionField = this.caseDetails.metadataFields.find(field => field.id === '[JURISDICTION]');
```

Comment:
can this '[JURISDICTION]' export to a const in common const file?

Reply:
Done